tools/cli/wskadmin

```diff
@@ -211,6 +211,7 @@ def getUserCmd(args, props):
         print 'Failed to get subject (%s)' % res.read().strip()
         return 1
 
+# This should be getSubjectFromDb
 def getSubjecFromDb(args, props):
```

Comment:
It's hard to know what's a public api, so I chose not to "just fix it". Also, it needed to be called out as more significant than the rest.